packages/@lwc/engine-core/src/3rdparty/snabbdom/types.ts

```diff
@@ -77,7 +77,7 @@ export interface VNodeData {
     className?: any;
     style?: any;
     classMap?: Classes;
-    styleMap?: VNodeStyle;
+    styles?: VNodeStyle;
```

Comment:
I'm open to naming this something else, e.g. `styleArray`. It's a bit confusing to have both `style` and `styles`. 

Although whatever we name it, that will literally be the compiled output, because it can't be mangled.

Reply:
`styleDecls`?

Reply:
Done. I like `styleDecls` as a name.